### PR TITLE
feat(arrs): add support for including unmonitored titles

### LINKED ISSUE #95: Allow unmonitored
I'd like to create filters for all items in arrs, but currently, there's no option to include unmonitored items. It would be great to have a configuration setting to allow this, this would also be beneficial for cross-seed filters.

### PR DESCRIPTION
**Pull request by s0up4200**
Closes #95

### CODE REVIEW

**Review by frant4 — APPROVED** [review]

File: internal/domain/config.go [hunk 1]
```diff
@@ -6,13 +6,13 @@ import (
...
 
-	"github.com/autobrr/omegabrr/internal/apitoken"
-
 	"github.com/knadh/koanf"
 	"github.com/knadh/koanf/parsers/yaml"
 	"github.com/knadh/koanf/providers/file"
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
+
+	"github.com/autobrr/omegabrr/internal/apitoken"
```
Comment (zze0s) [inline internal/domain/config.go:15]: Minor thing but we usually order imports like: stdlib, application, 3rd party

File: internal/processor/radarr.go [hunk 2]
```diff
@@ -6,14 +6,14 @@ import (
...
-	"github.com/autobrr/omegabrr/pkg/autobrr"
 	"github.com/pkg/errors"
-
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	"golift.io/starr"
 	"golift.io/starr/radarr"
+
+	"github.com/autobrr/omegabrr/internal/domain"
+	"github.com/autobrr/omegabrr/pkg/autobrr"
```
Comment (zze0s) [inline internal/processor/radarr.go:16]: Import order

File: internal/processor/service.go [hunk 3]
```diff
@@ -6,11 +6,11 @@ import (
...
 	"time"
 
-	"github.com/autobrr/omegabrr/internal/domain"
-	"github.com/autobrr/omegabrr/pkg/autobrr"
-
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
+
+	"github.com/autobrr/omegabrr/internal/domain"
+	"github.com/autobrr/omegabrr/pkg/autobrr"
```
Comment (zze0s) [inline internal/processor/service.go:13]: Import order.

File: internal/processor/sonarr.go [hunk 4]
```diff
@@ -6,14 +6,14 @@ import (
...
-	"github.com/autobrr/omegabrr/pkg/autobrr"
 	"github.com/pkg/errors"
-
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	"golift.io/starr"
 	"golift.io/starr/sonarr"
+
+	"github.com/autobrr/omegabrr/internal/domain"
+	"github.com/autobrr/omegabrr/pkg/autobrr"
```
Comment (zze0s) [inline internal/processor/sonarr.go:16]: Import order.

**Review by zze0s — CHANGES_REQUESTED** [review]
LGTM! Just some minor changes needed for the import order to keep it consistent.

**Review by zze0s — APPROVED** [review]

**Pull request merged**